CryptoTechReminderSystem.Test/SlackGatewayTests.cs

```diff
@@ -15,91 +17,151 @@ public class SlackGatewayTests
-
-            _fluentSimulator.Start();
-        }
-        
-        [TearDown]
-        public void TearDown()
-        {
-            _fluentSimulator.Stop();
-        }
-        
-        [Test]
-        public void CanSendAPostMessageRequest()
-        {
-            _slackGateway.Send(new Message());
+            [Test]
+            public void CanSendAPostMessageRequest()
+            {
+                _slackGateway.Send(new Message());
             
-            var receivedRequest = _fluentSimulator.ReceivedRequests.First();
+                var receivedRequest = _fluentSimulator.ReceivedRequests.First();
                 
-            receivedRequest.Url.Should().Be(PostMessageApiUrl);
-        }
-        
-        [Test]
-        public void CanSendAPostMessageRequestWithAToken()
-        {
-            _slackGateway.Send(new Message());
+                receivedRequest.Url.Should().Be(PostMessageApiUrl);
+            }
             
-            var receivedRequest = _fluentSimulator.ReceivedRequests.First();
+            [Test]
+            public void CanSendAPostMessageRequestWithAToken()
+            {
+                _slackGateway.Send(new Message());
+            
+                var receivedRequest = _fluentSimulator.ReceivedRequests.First();
                 
-            receivedRequest.Headers["Authorization"].Should().Be(
-                "Bearer " + Token
-            );
-        }
+                receivedRequest.Headers["Authorization"].Should().Be("Bearer " + Token);
+            }
         
-        [Test]
-        public void CanSendAMessageToAUser()
-        {
-            var channel = "U98DL811";
-            var message = new Message()
+            [Test]
+            public void CanSendAMessageToAUser()
             {
-                Channel = channel
-            };
+                var channel = "U98DL811";
+                var message = new Message()
+                {
+                    Channel = channel
+                };
             
-            _slackGateway.Send(message);
+                _slackGateway.Send(message);
             
-            var receivedRequest = _fluentSimulator.ReceivedRequests.First();
+                var receivedRequest = _fluentSimulator.ReceivedRequests.First();
             
-            DeserializeObject<PostMessageRequest>(receivedRequest.RequestBody).Channel.Should().Be(channel);
-
-        }
+                JObject.Parse(receivedRequest.RequestBody)["channel"].ToString().Should().Be(channel);
+            }
         
-        [Test]
-        public void CanSendAMessageToAUserWithAText()
+            [Test]
+            public void CanSendAMessageToAUserWithAText()
+            {
+                var text = "Please make sure your timesheet is submitted by 13:30 on Friday.";
+                var message = new Message()
+                {
+                    Channel = "U0112WTW",
+                    Text = text
```

Comment:
Is it better to have the test inline as you have done with the channel?

Reply:
We do not need the `Channel` property for this test as we are testing the `text` only.